betelgeuse.py

```diff
@@ -213,12 +217,15 @@ def test_case(path, collect_only, project):
                 # Ensure that a single match for the Test Case is
                 # returned.
                 assert len(results) == 1
-                test_case = results[0]
+                # Fetch the test case in order to get all of its
+                # fields and values.
+                test_case = TestCase(project, results[0].work_item_id)
```

Comment:
Sadly, I have to fetch the `TestCase` object in order to access its `caseautomation` value. I tried to figure out which field to fetch from `TestCase.query` but none of the values I tried (`'caseautomation'`, `'caseautomationstatus'`) seemed to work. If we can figure out how to fetch this field from a query, then we can move this "fetch" statement back inside the conditional statement.